docs/docsite/rst/playbooks_variables.rst

```diff
@@ -172,53 +172,55 @@ To see what information is available, try the following::
 
     ansible hostname -m setup
 
-This will return a ginormous amount of variable data, which may look like this, as taken from Ansible 1.4 on a Ubuntu 12.04 system::
+This will return a ginormous amount of variable data, which may look like this, as taken from Ansible 1.4 on a Ubuntu 12.04 system
+
+.. code-block: none
```

Comment:
This should be json. But there's several cleanups to the data that are needed.  I'll fix it after.